accounts/models.py

```diff
@@ -260,3 +260,17 @@ def create_user_profile(sender, instance, created, **kwargs):
         profile.save()
 
 post_save.connect(create_user_profile, sender=User)
+
+class DonationCampaign(models.Model):
+    goal = models.DecimalField(max_digits=6, decimal_places=2)
+    date_start = models.DateField()
+
+
+class Donation(models.Model):
+    email = models.CharField(max_length=255)
```

Comment:
This still doesn't allow for people to donate while logged in AND use their Freesound username...

I.e. imho donations should really work like the contact form:

```
if logged_in:
   donate as user or donate anonymously
else:
   give your name here or donate anonymously
```

@ffont your opinion?